workspaces/liferay-sample-workspace/client-extensions/liferay-sample-editor-config-contributor/src/index.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+interface IConfig {
+	[key: string]: any;
+}
+
+interface IEditorConfigTransformer {
+	(args: IConfig): IConfig;
+}
+
+const editorConfigTransformer: IEditorConfigTransformer = (config) => {
+	const toolbar: [string[]] = config.toolbar_liferay;
+
+	toolbar.push(['ImageSelector']);
+
+	return {
+		...config,
+		extraPlugins: 'itemselector',
```

Comment:
So, for example, if we added the `T` to the `js-api` PR (see https://github.com/liferay/liferay-frontend-projects/pull/1185#discussion_r1431018333) we could have types that described this return (`extraPlugins`, `toolbar_liferay` and such...)

Don't know though if it would be so useful and/or if we would be able or want to maintain such interfaces...

Reply:
We shall not maintain those, they must exist outside for importing by the CX implementors. Would be a very nice addition